### PR TITLE
gift card enable option

### PR DESCRIPTION
**Pull request by senthilkumar-muthusamy**
Fixed issue related to Heartland gift card

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ Contributors: globalpayments
 Tags: woocommerce, woo, unified, commerce, platform, global, payments, ucp, heartland, payment, systems, tsys, genius, 3DS, gateway, token, tokenize, save cards
 Requires at least: 5.4
 Tested up to: 5.8
```
Comment (MSmedal) [inline readme.txt:5]: After testing and prior to release we'll want to update this to reflect the current WC version

Comment (senthilkumar-muthusamy) [reply readme.txt]: updated the version details

**Review by MSmedal — COMMENTED** [review]
Everything looks great and works well in my ad hoc testing. Please bump the version number one more time, since there has been another release in the meantime, then we can merge.

**Comment by senthilkumar-muthusamy** [comment]
> Everything looks great and works well in my ad hoc testing. Please bump the version number one more time, since there has been another release in the meantime, then we can merge.

I updated the version number and resolved merge conflicts. Please review again

**Review by MSmedal — APPROVED** [review]

**Pull request merged**